src/zos_console/zowe/zos_console_for_zowe_sdk/console.py

```diff
@@ -27,7 +27,7 @@ class Console(SdkApi):
        A profile in dict (json) format
     """
 
-    def __init__(self, connection: dict):
+    def __init__(self, connection: dict, log = False):
         super().__init__(connection, "/zosmf/restconsoles/consoles/defcn", logger_name=__name__)
```

Comment:
We need to pass the `log` argument on to the SdkApi class. This comment applies to all the other SDKs too 😋 
```suggestion
        super().__init__(connection, "/zosmf/restconsoles/consoles/defcn", logger_name=__name__, log=log)
```